<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VendorMenu</class>
 <widget class="QMainWindow" name="VendorMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(214, 231, 243);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="allItems">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>621</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: white;
    border-radius: 10px;
}

QTableView QHeaderView::section {
    background-color: #4DA6FF;
    color: white;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
    padding: 5px;
}

QTableView QHeaderView::section:checked {
    background-color: #4DA6FF;
    color: white;
    border-radius: 10px;
    padding: 5px;
}

QScrollBar:vertical {
    background-color: #FFFFFF;
    border-radius: 10px;
    margin-right: 5px;
}

QScrollBar::handle:vertical {
    background-color: #4DA6FF;
    border-radius: 10px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QTableWidget::item {
    background-color: white;
    border-radius: 10px;
}

QTableWidget::item:selected {
    background-color: #4DA6FF;
    color: white;
}

QTableWidget::item:selected:focus {
    outline: none;
}

QTableWidget::item:selected:active {
    border-top-left-radius: 12px;
    border-bottom-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}

QTableWidget::item:selected:first {
    border-top-left-radius: 12px;
    border-bottom-left-radius: 12px;
}

QTableWidget::item:selected:last {
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}
</string>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Catagory</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sale Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>All Items</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="venderCart">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>90</y>
      <width>621</width>
      <height>551</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget {
    background-color: white;
    border-radius: 10px;
}

QTableView QHeaderView::section {
    background-color: #4DA6FF;
    color: white;
	border-top-left-radius: 10px;
	border-top-right-radius: 10px;
    padding: 5px;
}

QTableView QHeaderView::section:checked {
    background-color: #4DA6FF;
    color: white;
    border-radius: 10px;
    padding: 5px;
}

QScrollBar:vertical {
    background-color: #FFFFFF;
    border-radius: 10px;
    margin-right: 5px;
}

QScrollBar::handle:vertical {
    background-color: #4DA6FF;
    border-radius: 10px;
}

QScrollBar::add-line:vertical,
QScrollBar::sub-line:vertical {
    height: 0px;
}

QTableWidget::item {
    background-color: white;
    border-radius: 10px;
}

QTableWidget::item:selected {
    background-color: #4DA6FF;
    color: white;
}

QTableWidget::item:selected:focus {
    outline: none;
}

QTableWidget::item:selected:active {
    border-top-left-radius: 12px;
    border-bottom-left-radius: 12px;
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}

QTableWidget::item:selected:first {
    border-top-left-radius: 12px;
    border-bottom-left-radius: 12px;
}

QTableWidget::item:selected:last {
    border-top-right-radius: 12px;
    border-bottom-right-radius: 12px;
}
</string>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::DashLine</enum>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Sale Price</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="aprove">
    <property name="geometry">
     <rect>
      <x>1182</x>
      <y>678</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Aprove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reload">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>678</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Reload</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deny">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>678</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Deny</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalNo">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>677</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>680</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Total</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalPrice">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>677</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border-radius: 15px;
    background-color: white;
	padding-left: 10px;
    padding-right: 10px; 
	font-size: 14px;
 	color: #4da6ff;
    font-weight: bold;
	text-align: center;
}

QLineEdit:focus, QLineEdit:hover {
    border: 1px solid #4da6ff;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>680</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logOut">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>10</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #2196F3;
    color: #FFFFFF;
    border-radius: 14px;
    padding: 4px 10px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #1976D2;
}

QPushButton:pressed {
    background-color: #0D47A1;
}
</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="emailLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>421</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:32px;
}
</string>
    </property>
    <property name="text">
     <string>email</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
	font-size:32px;
}
</string>
    </property>
    <property name="text">
     <string>name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    color: #2196F3;
    font-weight: bold;
}
</string>
    </property>
    <property name="text">
     <string>Requested by Admin</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
